src/main/java/duke/tasks/Task.java

```diff
@@ -0,0 +1,61 @@
+package duke.tasks;
+/**
+ * This represents a Task object for the list.
+ * 
+ * @author Rishabh Anand
+ * @version CS2103, AY21/222 Semester 1
+ * 
+ */
+abstract public class Task {
+    protected String description;
+    protected Boolean status;
```

Comment:
Consider using the primitive type `boolean` instead for performance and ease of coding